docs/manual/tags.md

```diff
@@ -344,259 +182,189 @@ denotes the scriptlet interpreter and possible arguments, and flags tag
+Preun      | 1025 | string
+Preunflags | 5022 | int32
+Preunprog  | 1087 | string array
 
-* Dsaheader (267) bin
+`%verify` script is executed when the package is verified (e.g. with `rpm -V`)
 
-  OpenPGP DSA signature of the header (if thus signed)
+Verifyscript      | 1079 | string
+Verifyscriptflags | 5026 | int32
+Verifyscriptprog  | 1091 | string array
 
-* Longsigsize (270) int64
-
-  Header+payload size if > 4GB.
-
-* Payloaddigest (5092) string array
-
-  Cryptographic digest of the compressed payload.
-
-* Payloaddigestalgo (5093) int32
-
-  ID of the payload digest algorithm.
-
-* Payloaddigestalt (5097) string array
-
-  Cryptographic digest of the uncompressed payload.
-
-* Rsaheader (268) bin
-
-  OpenPGP RSA signature of the header (if thus signed).
-
-* Sha1header (269) string
-
-  SHA1 digest of the header.
-
-* Sha256header (273) string
-
-  SHA256 digest of the header.
-
-* Siggpg (262) bin
+### Triggers
 
-  OpenPGP DSA signature of the header+payload (if thus signed).
+[Triggers](triggers.md) are stored in a combination of dependency and scriptlet tags.
 
-* Sigmd5 (261) bin
+Triggerscripts, Triggerscriptflags and Triggerscriptprog form a scriptlet
+triplet, and Triggername, Triggerflags, Triggerversion form a dependency
+triplet. Triggerindexes is a per dependency index into Triggerscripts,
+binding the two triplets together.
 
-  MD5 digest of the header+payload.
+Triggerflags       | 1068 | int32 array
+Triggerindex       | 1069 | int32 array
+Triggername        | 1066 | string array
+Triggerscriptflags | 5027 | int32 array
+Triggerscriptprog  | 1092 | string array
+Triggerscripts     | 1065 | string array
+Triggerversion     | 1067 | string array
 
-* Sigpgp (259) bin
+### File triggers
 
-  OpenPGP RSA signature of the header+payload (if thus signed).
+[File trigger](file_triggers.md) tags are like normal trigger tags,
+with an additional priority tag to affect trigger running order. File
+triggers in Filetrigger-tags run once per triggered package, whereas
+triggers in Transfiletrigger-tags run once per transaction.
+
+Filetriggerflags       | 5072 | int32 array
+Filetriggerindex       | 5070 | int32 array
+Filetriggername        | 5069 | string array
+Filetriggerpriorities  | 5084 | int32 array
+Filetriggerscriptflags | 5068 | int32 array
+Filetriggerscriptprog  | 5067 | string array
+Filetriggerscripts     | 5066 | string array
+Filetriggerversion     | 5071 | string array
+
+Transfiletriggerflags       | 5082 | int32 array
+Transfiletriggerindex       | 5080 | int32 array
+Transfiletriggername        | 5079 | string array
+Transfiletriggerpriorities  | 5085 | int32 array
+Transfiletriggerscriptflags | 5078 | int32 array
+Transfiletriggerscriptprog  | 5077 | string array
+Transfiletriggerscripts     | 5076 | string array
+Transfiletriggerversion     | 5081 | string array
 
-* Sigsize (257) int32
+## Signatures and digests
 
-  Header+payload size.
+[Signatures](signatures.md) allow to verify the origin of a package.
```

Comment:
Not all of these tags are actually signatures.  There should be a separation between cryptographic signatures, and entries that just happen to be in the signature header.